### DISCUSSION TITLE
How change default language?

### DISCUSSION THREAD

**Discussion by Dm1tr1ch** [nocodb/nocodb#6104] [Q&A]
Hello,

Can you help me with changing default language?

**Comment by wingkwong** [comment]
You can only do it via UI.

My Project
![image](https://github.com/nocodb/nocodb/assets/35857179/13cc39a5-fac4-4e06-85b8-46e929585aa2)

Inside Project
![image](https://github.com/nocodb/nocodb/assets/35857179/7c94cb18-e41a-413a-9942-580bb3a04df3)

**Reply by Dm1tr1ch** [reply]
I want default language for all current and any new projects. I need change "system" default language. Now, users should change language when first time login or if they clear coockies.
